### ISSUE TITLE
An error occurs when coc#float#scroll() is run by press the mapped key in normal mode

### ISSUE THREAD

**Issue by hexium310** [neoclide/coc.nvim#2727]
## Result from CocInfo

```
## versions

vim version: NVIM v0.5.0-dev+948-g2debabb08
node version: v15.4.0
coc.nvim version: 0.0.79-e444c57756
coc.nvim directory: /Users/hexin/.cache/dein/repos/github.com/neoclide/coc.nvim_release
term: iTerm.app
platform: darwin

## Log of coc.nvim

2020-12-17T23:15:16.399 INFO (pid:26890) [services] - registered service "vimlsp"
2020-12-17T23:15:16.402 INFO (pid:26890) [services] - vim language server state change: stopped => starting
2020-12-17T23:15:16.406 INFO (pid:26890) [plugin] - coc.nvim 0.0.79-e444c57756 initialized with node: v15.4.0 after 224ms
2020-12-17T23:15:16.416 INFO (pid:26890) [language-client-index] - vimlsp started with 26892
2020-12-17T23:15:16.650 INFO (pid:26890) [services] - vim language server state change: starting => running
2020-12-17T23:15:16.657 INFO (pid:26890) [services] - service vimlsp started
```

## Describe the bug

The following error occurs when coc#float#scroll() is run by press the mapped key in normal mode. This issue occurs in only Neovim.

```
Error detected while processing function coc#float#scroll[10]..coc#float#scroll_win[21]..<SNR>31_win_setview[2]..coc#compat#execute:
line   12:
E5555: API call: E523: Not allowed here
```

## Reproduce the bug

**We will close your issue when you don't provide minimal vimrc and we can't
reproduce it**

- Create file `mini.vim` with：

```vim
set nocompatible
set runtimepath^=~/.cache/dein/repos/github.com/neoclide/coc.nvim_release
filetype plugin indent on
syntax on
set hidden

nnoremap <silent><nowait><expr> <C-f> coc#float#has_scroll() ? coc#float#scroll(1) : "\<C-f>"
nnoremap <silent><nowait><expr> <C-b> coc#float#has_scroll() ? coc#float#scroll(0) : "\<C-b>"

" The code for trial 
function! _()
  call nvim_set_client_info()
endfunction
```

- Start Neovim and open `mini.vim` with command: `nvim -u mini.vim mini.vim`

- Operate vim.
  1. Move the cursor over `nvim_set_client_info` (or a function that displayed description is long)
  2. Run `:call CocActionAsync('doHover')` to open the floating window
  3. Press `<C-f> to scroll the floating window

**Comment by fannheyward** [comment]
Can't reproduce with nvim v0.5.0-dev+941-g9c56d8e5f and latest coc.

**Comment by chemzqm** [comment]
Fixed by afb79f1
